test/Microsoft.DotNet.ProjectJsonMigration.Tests/GivenAProjectMigrator.cs

```diff
@@ -77,6 +77,32 @@ public void It_has_error_when_migrating_a_non_csharp_app()
             errorMessage.Should().Contain("MIGRATE20013::Non-Csharp App: Cannot migrate project");
         }
 
+        [Fact]
+        public void ItHasErrorWhenMigratingAProjectJsonWithoutAFrameworks()
+        {
+            var testAppName = "TestLibraryWithProjectFileWithoutFrameworks";
```

Comment:
I don't think we need a var here, I'd just inline the name so we don't need to reason about it in the test code.